Install/Install.cmake

```diff
@@ -11,23 +11,19 @@ function(install_targets)
     foreach(target IN LISTS ARGN)
         get_target_property(target_type ${target} TYPE)
         if(NOT target_type STREQUAL "INTERFACE_LIBRARY")
-            install(FILES $<TARGET_FILE_DIR:${target}>/${target}.pdb DESTINATION lib OPTIONAL)
+            install(FILES "$<TARGET_FILE_DIR:${target}>/${target}.pdb" DESTINATION lib OPTIONAL)
         endif()
     endforeach()
 endfunction()
 
-function(install_include)
-    install(DIRECTORY ${ARGV0} TYPE INCLUDE)
-endfunction()
-
-function(install_include_for_target)
-    get_target_property(TARGET_INCLUDE_DIRECTORIES ${ARGV0} INCLUDE_DIRECTORIES)
-    foreach(include_directory IN LISTS TARGET_INCLUDE_DIRECTORIES)
-        if(${ARGC} STREQUAL "1")
-            install_include("${include_directory}")
-        else()
-            install_include("${include_directory}/${ARGV1}")
-        endif()
+function(install_include_for_targets)
+    foreach(target IN LISTS ARGN)
```

Comment:
Is that loop necessary?

Reply:
No, but it's more consistent with the other functions.